content/glass_elevator/rfc/_index.md

```diff
@@ -0,0 +1,40 @@
++++
+type = "article"
+title = "Request for Comments (RFC)"
+date = "2006-12-18T21:07:21.000Z"
+tags = []
++++
+
+## This is an index of all Request for Comments (RFC) documents submitted to the Glass Elevator project up until 2017.
+
+- [3D Accelerated Haiku Desktop](3d_accelerated_haiku_desktop/)
+- [Against Directory](against_directories/)
+- [BFS2 or Our Next FS](bfs2_or_our_next_fs/)
+- [Calculated Attributes](calculated_attributes/)
+- [Cross Referenced Files](cross_referenced_files/)
+- [Design Goals For the Game Kit](design_goals_for_the_game_kit/)
+- [DevFS Attributes](devfs_attributes/)
+- [Download Server](download_server/)
+- [Drag and Drop: Better Visualization to Discern Between Copying Moving and Linking](drag_and_drop_better_visualization_to_discern_between_copying_moving_linking/)
+- [Drag and Drop Save](drag_and_drop_save/)
+- [Feature Wishlist](feature_wishlist/)
+- [Fildirute](fildirute/)
+- [File User Polymorphism](file_user_polymorphism/)
+- [Fusing Tabs](fusing_tabs/)
+- [Inovative Concepts Midia Overal Haiku Design](inovative_concepts_media_overal_haiku_design/)
```

Comment:
```suggestion
- [Inovative Concepts Media Overall Haiku Design](inovative_concepts_media_overal_haiku_design/)
```

Reply:
Corrected this and several other grammatical mistakes in next commit.